card/card.css

```diff
@@ -0,0 +1,152 @@
+body{
+    margin-bottom: 5rem;
+}
+
+img{
+    width:100%;
+    height: auto;
+}
+
+.basic-card{
+    margin-left: 5rem;
+    position: relative;
+    display: flex;
+    box-shadow: 0 0 5px 1px var(--light-grey);
+}
+
+.basic-card-vertical{
+    width:20rem;
+    flex-direction: column;
+}
+
+.basic-card-horizontal{
+    flex-direction: row;
+    width: 40rem;
+    align-items:center; 
+}
+
+.img-section{
+    position:relative;
+    flex:1;
+}
+
+.card-body{
+    margin: 1rem;
+    flex: 1;
+} 
+
+.price-details{
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding:5px;
+}
+
+.card-body p,
+.old-price,.offer{
+    color:var(--grey);
+}
+
+.new-price,.offer{
+    font-weight: bold;
+}
+
+.new-price{
+    font-size: 1.5rem;
+}
+
+.old-price{
+    text-decoration: line-through;
+}
+
+.rating{
+    background-color: var(--green);
+    display: flex;
+    width: 4rem;
+    justify-content: space-evenly;
+    align-items: center;
+    border-radius: 1rem;
+    padding:5px; 
+    color: var(--white);
+}
+
+.rating i{
+    font-size: 0.8rem;
+}
+
+.btn-wishlist{
+    position: absolute;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: transparent;
+    font-size: 1.6rem;
+    top:5px;
+    right:1rem;
+    color:var(--light-red2);
+}
+
+.btn-wishlist:hover{
+    color:var(--light-red3);
+}
+
+.btn-addtocart{
+    margin-top:8px;
```

Comment:
try to give space after css property, makes it easier to read the values.

Reply:
Yes, I had made changes